public/locations-list-map.js

```diff
@@ -195,6 +201,17 @@ function ctn(text) {
   return document.createTextNode(text);
 }
 
+// Turns a string with embedded \n characters into an Array of text nodes separated by <br>
+function multilineStringToNodes(input) {
+  const textNodes = input.split('\n').map((s) => ctn(s));
+  let returnedNodes = [];
+  textNodes.forEach((e) => {
+    returnedNodes.push(e);
+    returnedNodes.push(document.createElement('br'));
+  })
+  return returnedNodes.slice(0,-1);
```

Comment:
You can just decrement the length of returnedNodes to chop off the last <br>, instead of making a copy via slice()